pkg/webhook/webhook.go

```diff
@@ -0,0 +1,76 @@
+package webhook
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strconv"
+	"time"
+
+	jsonpatchv2 "gomodules.xyz/jsonpatch/v2"
+	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
+	"sigs.k8s.io/controller-runtime/pkg/webhook"
+	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+
+	nmstatev1alpha1 "github.com/nmstate/kubernetes-nmstate/pkg/apis/nmstate/v1alpha1"
+)
+
+var log = logf.Log.WithName("webhook")
+
+func resetConditionsPatch() jsonpatchv2.Operation {
+	return jsonpatchv2.Operation{
+		Path:      "/status/conditions",
+		Operation: "replace",
+		Value:     nmstatev1alpha1.ConditionList{},
+	}
+
+}
+
+func mutationAnnotationPatch(annotations map[string]string) jsonpatchv2.Operation {
+	key := "conditions_mutation"
```

Comment:
Please make it namespaced so it is obvious it was added by us. Also, AFAIK I'd make it lowerCamelCase. Also, if it cold be more specific, spelling that it is webhookMutationTimestamp maybe?

Reply:
Reading best practices from here [1] I am going to name it nmstate.io/webhook-mutation-timestamp

[1] https://www.replex.io/blog/9-best-practices-and-examples-for-working-with-kubernetes-labels